File: pyRabbitMQ/send.py
#!/usr/bin/env python
import pika

# Since we're using default port, we don't have to adjust
connection = pika.BlockingConnection(pika.ConnectionParameters('localhost'))
channel = connection.channel()

# Create queue  to make sure the recipient exist
channel.queue_declare(queue='hello')

channel.basic_publish(exchange='',
                      routing_key='hello',
                      body='Hello World from Python!')
print(" [x] Sent 'Hello World from Python!'")

connection.close()

File: phpRabbitMQ/send.php
<?php

require_once __DIR__ . '/vendor/autoload.php';
use PhpAmqpLib\Connection\AMQPStreamConnection;
use PhpAmqpLib\Message\AMQPMessage;

$connection = new AMQPStreamConnection('172.17.0.2', 5672, 'guest', 'guest');
$channel = $connection->channel();

$channel->queue_declare('hello', false, false, false, false);

$msg = new AMQPMessage('Hello World using PHP!');
$channel->basic_publish($msg, '', 'hello');

echo " [x] Sent 'Hello World using PHP!'\n";

$channel->close();
$connection->close();
?>